datalad/support/tests/test_annexrepo.py

```diff
@@ -1204,7 +1205,10 @@ def test_annex_ssh(topdir):
     # copy to the new remote:
     ar.copy_to(["foo"], remote="ssh-remote-2")
 
-    ok_(exists(socket_2))
+    if not exists(socket_2):
```

Comment:
lets add `  # pragma: no cover` to both cases so there is no jumps in coverage depending on the git-annex-weather ;-)